x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -496,6 +496,139 @@ export const getPrepopulatedRansomwareException = ({
   };
 };
 
+export const getPrepopulatedMemorySignatureException = ({
+  listId,
+  ruleName,
+  eventCode,
+  listNamespace = 'agnostic',
+  alertEcsData,
+}: {
+  listId: string;
+  listNamespace?: NamespaceType;
+  ruleName: string;
+  eventCode: string;
+  alertEcsData: Flattened<Ecs>;
+}): ExceptionsBuilderExceptionItem => {
+  const { process } = alertEcsData;
+  return {
+    ...getNewExceptionItem({ listId, namespaceType: listNamespace, ruleName }),
+    entries: addIdToEntries([
+      {
+        field: 'Memory_protection.feature',
+        operator: 'included',
+        type: 'match',
+        value: alertEcsData.Memory_protection?.feature ?? '',
```

Comment:
Q: Do we have to add an entry with empty value or should we just not add a new entry if we don't have the specific value?

Reply:
This should always come with a value.

Reply:
Then I'm wondering why this key can be undefined `alertEcsData.Memory_protection?.feature ?? ''` 
That seems `.feature` can be null and then value will be empty.
If this is ok, forget this comment 🙂

Reply:
@dasansol92 there's not really a good reason for this type to allow undefined. But I saw that every single type inside the ECS is marked as possible undefined and I followed that pattern. Based on the real data I got for this kind of alarm, they should not be undefined and the other exception implementations do the same as me here.

It actually makes more sense to conditionally add fields only when they are populated rather than this than this nullish coalescing.

I'll keep it like this to follow the pattern and I'll reevaluate it when I start working in the behaviour protection that should also touch this area.